addons/website/views/snippets/s_list_group.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_list_group" name="List Group">
+    <div class="s_list_group my-3" style="--s_list_group-icon-bg: color-mix(in srgb, currentColor 10%, transparent);">
+        <ul>
+            <li>Wonderful experience</li>
+            <li>Quick support</li>
+            <li>Access to our latest content</li>
+        </ul>
+    </div>
+</template>
+
+<record id="website.s_list_group_000_scss" model="ir.asset">
+    <field name="name">ListGroup 000 SCSS</field>
+    <field name="bundle">web.assets_frontend</field>
+    <field name="path">website/static/src/snippets/s_list_group/000.scss</field>
+</record>
```

Comment:
We don't use `<record>` anymore for these assets, but `<asset>` :+1: 
So change this to :arrow_down: 
```xml
<asset id="website.s_list_group_000_scss" name="List Group 000 SCSS">
    <bundle>web.assets_frontend</bundle>
    <path>website/static/src/snippets/s_list_group/000.scss</path>
</asset>
```